tests/test_save_load.py

```diff
@@ -233,6 +233,23 @@ def test_exclude_include_saved_params(tmp_path, model_class):
     os.remove(tmp_path / "test_save.zip")
 
 
+def test_save_load_pytorch_var(tmp_path):
+    model = SAC("MlpPolicy", "Pendulum-v0", seed=3, policy_kwargs=dict(net_arch=[64], n_critics=1))
+    model.learn(200)
+    model.save(str(tmp_path / "sac_pendulum"))
```

Comment:
Super nit, but inclined to split this as:
```suggestion
    save_path = str(tmp_path / "sac_pendulum")
    model.save(save_path)
```
and then reuse `save_path` on line 245 below